.gitignore

```diff
@@ -126,3 +126,6 @@ metadata-service/war/bin/
 metadata-utils/bin/
 test-models/bin/
 
+datahub-executor/
+datahub-integrations-service/
+metadata-ingestion-modules/
```

Comment:
Think this is fine to include? Tired of seeing these directories when switching repos

Reply:
should be `metadata-ingestion-modules/acryl-cloud`